apps/etterlatte-saksbehandling-ui/client/src/components/behandling/brevutfall/BrevutfallVisning.tsx

```diff
@@ -51,16 +63,48 @@ export const BrevutfallVisning = (props: {
             <BodyShort>{brevutfallOgEtterbetaling.etterbetaling ? 'Ja' : 'Nei'}</BodyShort>
           </VStack>
           {brevutfallOgEtterbetaling.etterbetaling && (
-            <HStack gap="8">
-              <VStack gap="2">
-                <Label>Fra og med</Label>
-                <BodyShort>{formaterMaanedDato(brevutfallOgEtterbetaling.etterbetaling.datoFom!!)}</BodyShort>
-              </VStack>
-              <VStack gap="2">
-                <Label>Til og med</Label>
-                <BodyShort>{formaterMaanedDato(brevutfallOgEtterbetaling.etterbetaling.datoTom!!)}</BodyShort>
-              </VStack>
-            </HStack>
+            <>
+              <HStack gap="8">
+                <VStack gap="2">
+                  <Label>Fra og med</Label>
+                  <BodyShort>{formaterMaanedDato(brevutfallOgEtterbetaling.etterbetaling.datoFom!!)}</BodyShort>
+                </VStack>
+                <VStack gap="2">
+                  <Label>Til og med</Label>
+                  <BodyShort>{formaterMaanedDato(brevutfallOgEtterbetaling.etterbetaling.datoTom!!)}</BodyShort>
+                </VStack>
+              </HStack>
+              {sakType == SakType.BARNEPENSJON && (
+                <>
+                  <VStack gap="2">
+                    <Label>Er det krav i etterbetalingen?</Label>
+                    <BodyShort>{brevutfallOgEtterbetaling.etterbetaling?.inneholderKrav ? 'Ja' : 'Nei'}</BodyShort>
+                  </VStack>
+                  <VStack gap="2">
+                    <Label>Har bruker meldt inn frivillig skattetrekk?</Label>
+                    <BodyShort>
+                      {brevutfallOgEtterbetaling.etterbetaling?.frivilligSkattetrekk ? 'Ja' : 'Nei'}
+                    </BodyShort>
+                  </VStack>
+                  <VStack gap="2">
+                    <Label>Hvor mange måneder etterbetales det for?</Label>
+                    <BodyShort>
+                      {etterbetalingPeriodeValgToString(
+                        brevutfallOgEtterbetaling.etterbetaling?.etterbetalingPeriodeValg
+                      )}
+                    </BodyShort>
+                  </VStack>
+                  {brevutfallOgEtterbetaling.etterbetaling?.frivilligSkattetrekk && (
+                    <VStack gap="2">
+                      <Label>Er det lagt inn til og med dato på skattetrekk?</Label>
+                      <BodyShort>
+                        {brevutfallOgEtterbetaling.etterbetaling?.skatteTrekkFomTomDatoSatt ? 'Ja' : 'Nei'}
+                      </BodyShort>
+                    </VStack>
+                  )}
+                </>
```

Comment:
Kanskje ha en egen håndtering for at disse dataene ikke fins på historiske behandlinger? Så hvis det er undefined og vi ikke er i redigeringsmodus ikke vise det f.eks.?

Reply:
Godt poeng 👍

Reply:
[8392f06](https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/5273/commits/8392f064d8c7bcd29333cf14ebb1fb1957c2e54a)